app/models/incident_report.py

```diff
@@ -11,6 +11,10 @@ class Location(db.Model):
     incident_report_id = db.Column(db.Integer,
                                    db.ForeignKey('incident_reports.id'))
 
+    def __repr__(self):
+        # TODO: Show address instead?
```

Comment:
What's the benefit of including `original_user_text`? I think this is fine and not worth putting off the merge. If someone wants a more useful repr, they can add it in their own PR. This currently isn't used anywhere anymore, I'm just leaving it in for future convenience so others can easily modify it for their own uses.